### DISCUSSION TITLE
Change Screen Resolution on Ubuntu Latest with Github Actions

### DISCUSSION THREAD

**Discussion by gokultw** [actions/virtual-environments#5943] [General questions] [answered]
I want to run my E2E tests using Github actions with Ubuntu Latest.
Scripts are failing due to low resolution and I want to increase the same. Please let me know how to do it.

Looking for solution like something which is already available for Mac: - https://github.com/actions/virtual-environments/issues/393

**Comment by miketimofeev** [comment, marked as answer]
@gokultw Ubuntu images don't have any GUI so the way probably depends on the software you use to run tests

**Reply by gokultw** [reply]
Thanks for your reply, will look for changes in my E2E framework.

**Comment by SinSkylark** [comment]
@gokultw did your issue get resolved? Are you perhaps using xvfb or xvfb-run?

**Reply by gokultw** [reply]
Yes the issue is resolved, I'm using `xvfb-run`

**Reply by gladykov** [reply]
@gokultw Could you share some example how you were able to do it?

**Comment by rene-leanix** [comment]
> I want to run my E2E tests using Github actions with Ubuntu Latest. Scripts are failing due to low resolution and I want to increase the same. Please let me know how to do it.

@gokultw, in our case, we had Cypress e2e test failures because one of the libraries we were using was checking the value for `screen.width`. Overriding it in the test's before hook did the trick:

```ts
Object.defineProperty(window.screen, 'width', { get: () => 2560 });
```

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
